coderd/notifications/testdata/rendered-templates/TemplateUserRequestedOneTimePasscode-title.md.golden

```diff
@@ -0,0 +1 @@
+Your one-time passcode is enclosed.
```

Comment:
nit: alternatives:

Your One-Time Passcode for Coder
Your One-Time Passcode Inside
Your Coder Passcode: Use Within N Minutes